client/src/main/java/io/getstream/chat/android/client/errors/ChatError.kt

```diff
@@ -1,8 +1,8 @@
 package io.getstream.chat.android.client.errors
 
-open class ChatError(
-    val message: String?,
-    val cause: Throwable? = null
+public open class ChatError(
```

Comment:
Do we want clients to subclass this? If not, this would be a good time to make it `sealed` (or convert it into an `interface`, even).

Reply:
Because we have our own inheritor `ChatNetworkError` that used a lot in LLC

Reply:
Oh, and we need to be able to create regular `ChatError` instances as well, so we can't make it `sealed`, because then it would be `abstract`. Alright, thanks! `open` it is then 😄